### ISSUE TITLE
No way to specify custom ssh port

### ISSUE THREAD

**Issue by rothomp3** [louisdh/terminal#146]
As far as I can tell, the ssh command doesn't recognize any command-line options at all. It treats any argument as a host to connect to. This means I can't pass the `-p` flag to change the port from the default 22. Am I missing something?

**Comment by ddfznt** [comment]
it seems that the only flag he accepts is -q https://github.com/holzschu/ios_system/blob/5f4ac24955ed4270543de141cf85671c0ffb6787/ssh_main.c#L352

**Comment by unpin** [comment]
Same here.

**Comment by rothomp3** [comment]
Hmm I know more than enough C to fix that, if I had the free time 😓. Bleh, now I feel guilty haha

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by germanlarrosa** [comment]
Yup, faced same issue here. Any news about a possible fix? Thanks

**Comment by nathdrey** [comment]
Hi everybody, is there any update on this topic ?
